### DISCUSSION TITLE
Add Network Protocols List

### DISCUSSION THREAD

**Discussion by D3stinyDev** [CorentinTh/it-tools#1414] [Ideas]
I am currently working with Network Protocols, and I had to search the internet for a while, to find cheat sheets for those Network Protocols, and I couldn't even search for them.

So I wondered if IT-Tools.tech could add this feature, a basic list with a lot of Network Protocols, and if you click the Network Protocols, then it opens up a modal, with information about this Network Protocol.

**Comment by sharevb** [comment]
Hi @D3stinyDev, kind of a start here : https://github.com/CorentinTh/it-tools/pull/1029

**Comment by sharevb** [comment]
Hi @D3stinyDev , if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Reply by D3stinyDev** [reply]
Thank you,

I have found a lot of good tools, that I can use.

**Reply by sharevb** [reply]
Hi, could you quote this tools as may be cool be integrated in it-tools too ? thanks
